src/components/MultiHopTrade/components/SettingsPopover.tsx

```diff
@@ -0,0 +1,45 @@
+import type { FC } from 'react'
+import { memo, useCallback } from 'react'
+
+import { SharedSettingsPopover } from './SharedTradeInput/SharedSettingsPopover'
+
+import { selectUserSlippagePercentage } from '@/state/slices/tradeInputSlice/selectors'
+import { tradeInput } from '@/state/slices/tradeInputSlice/tradeInputSlice'
+import {
+  selectDefaultSlippagePercentage,
+  selectQuoteSlippageTolerancePercentage,
+} from '@/state/slices/tradeQuoteSlice/selectors'
+import { useAppDispatch, useAppSelector } from '@/state/store'
+
+type SettingsPopoverProps = {
+  isDisabled?: boolean
+  tooltipTranslation?: string
+}
+
+export const SettingsPopover: FC<SettingsPopoverProps> = memo(
```

Comment:
Do we really need to use `memo()`?

Reply:
I mean this is really just a copy of the previous `SlippagePopover` but I don't see why not hey!